internal/util/vault.go

```diff
@@ -208,130 +211,10 @@ func (kms *VaultKMS) SavePassphrase(key, value string) error {
 
 // DeletePassphrase deletes passphrase from Vault.
 func (kms *VaultKMS) DeletePassphrase(key string) error {
-	vaultToken, err := kms.getAccessToken()
+	err := kms.secrets.DeleteSecret(filepath.Join(kms.vaultPassphrasePath, key), kms.keyContext)
```

Comment:
When `filepath.Join(kms.vaultPassphrasePath, key)` file will be deleted?

Reply:
Unsure what you mean? This deletes the secret that is identified with the `filepath.Join(kms.vaultPassphrasePath, key)`. It is just the API replacement of doing the manual HTTP DELETE call.

Reply:
Ok, it seems that the secret gets deleted, but is not destroyed by Vault (it becomes empty). Not sure when the deleted secrets get completely removed though. For this, I have updated the e2e test that checks the contents of the secret.